### DISCUSSION TITLE
Can kill SONY TV  but can not kill smartphone.

### DISCUSSION THREAD

**Discussion by getlone** [elmoiv/elmocut#56] [Q&A]
Yes it show this but the phone still use net work normally. (It kill TV SONY BRAVIA properly tho .)

![Screenshot (717)](https://user-images.githubusercontent.com/66230020/221164921-03cb4189-babd-49bb-9641-2b84b2a9babf.jpg)

**Comment by elmoiv** [comment]
I couldn't reproduce the same issue on my side. Please try with different phones. If this issue occurs on all phones I will try to find a solution for it. If not, it may be an ARP spoofing protection feature in the phone.
